econtext/src/main/java/com/adyen/checkout/econtext/internal/ui/view/EContextView.kt

```diff
@@ -175,7 +176,8 @@ internal class EContextView @JvmOverloads constructor(
                 val country = countryAdapter?.getItem(position) ?: return@OnItemClickListener
                 onCountrySelected(country)
             }
-            countries.firstOrNull()?.let {
+            val initialCountry = countries.firstOrNull { it.isoCode == Locale.JAPAN.country } ?: countries.firstOrNull()
```

Comment:
Is there a way to not hardcode `Locale.JAPAN`, but get it dynamically based on the payment method type?

Reply:
Resolving it for now to not keep the PR waiting, if an update happens I will do it in a different PR.